app/src/test/java/org/groundplatform/android/persistence/local/LocalLocationOfInterestStoreTest.kt

```diff
@@ -307,35 +187,6 @@ class LocalDataStoreTests : BaseHiltTest() {
     }
   }
 
-  @Test
-  fun testGetOfflineAreas() = runWithTestDispatcher {
-    localOfflineAreaStore.insertOrUpdate(TEST_OFFLINE_AREA)
-    localOfflineAreaStore.offlineAreas().test {
-      assertThat(expectMostRecentItem()).isEqualTo(listOf(TEST_OFFLINE_AREA))
-    }
-  }
-
-  @Test
-  fun testParseVertices_emptyString() {
-    assertThat(parseVertices("")).isEqualTo(listOf<Any>())
-  }
-
-  @Test
-  fun testFormatVertices_emptyList() {
-    assertThat(formatVertices(listOf())).isNull()
-  }
-
-  @Test
-  fun testTermsOfServiceAccepted() {
-    localValueStore.isTermsOfServiceAccepted = true
-    assertThat(localValueStore.isTermsOfServiceAccepted).isTrue()
-  }
-
-  @Test
-  fun testTermsOfServiceNotAccepted() {
-    assertThat(localValueStore.isTermsOfServiceAccepted).isFalse()
-  }
```

Comment:
Are these tests covered somewhere else too? I see that the test coverage has dropped.

Reply:
This PR is only shifting of test cases for one file to another,

Reply:
Fixed now